src/sentry/replays/endpoints/project_replay_recording_segment_details.py

```diff
@@ -1,14 +1,14 @@
+from io import BytesIO
```

Comment:
i wonder if we should just delete this endpoint since segment_index is always used in the product. thoughts?

Reply:
@JoshFerge I agree.  I think we can drop it.  However, because this is a large change I think it would be better to do on a different pull request.